test/e2e/common/node/lifecycle_hook.go

```diff
@@ -253,6 +253,254 @@ var _ = SIGDescribe("Container Lifecycle Hook", func() {
 	})
 })
 
+var _ = SIGDescribe("[NodeAlphaFeature:SidecarContainers][Feature:SidecarContainers] Restartable Init Container Lifecycle Hook", func() {
```

Comment:
these tests have been flaking, with at least one of them failing in about 50% of the runs since they merged:

https://testgrid.k8s.io/presubmits-kubernetes-nonblocking#pull-kubernetes-e2e-gce-cos-alpha-features&include-filter-by-regex=NodeAlphaFeature:SidecarContainers|Test&width=20&show-stale-tests=